app/routes/$lang.categories._index.tsx

```diff
@@ -0,0 +1,54 @@
+import { Link, useLoaderData, useParams } from "@remix-run/react"
+import { json, LoaderFunction } from "@remix-run/node"
+import { Category, getCategories } from "~/sanity/queries"
+import { client } from "~/sanity/client"
+import { SupportedLanguages } from "~/i18n"
+import isLangSupportedLang from "~/sanity/queries/isLangSupportedLang"
+import { Layout } from "~/components/Layout"
+import { Typography } from "~/components/Typography"
+import { zeroWidthTrim } from "~/lib/zeroWidthTrim"
+
+interface IndexLoaderData {
+  categories: Category[]
+}
+
+export const loader: LoaderFunction = async ({ params }) => {
+  isLangSupportedLang(params.lang)
+  const categories = await getCategories(client, params.lang!)
+
+  return json<IndexLoaderData>({
+    categories,
+  })
+}
+
+export default function CategoriesRoute() {
+  const { categories } = useLoaderData() as IndexLoaderData
+  const params = useParams()
+  const lang = params.lang as SupportedLanguages
+
+  return (
+    <Layout
+      translationUrl={lang === "en" ? "/fr/categories" : "/en/categories"}
+      useMargins
+    >
+      <div>
+        {categories?.map((category) => {
+          const linkTo = zeroWidthTrim(
```

Comment:
were you noticing weird zero width characters on the string for `category.name[lang]`? That's why i added this zeroWidthTrim() function, might not be needed here unless it was happening here too

Reply:
i spent like two hours trying to track it down, ended up just saying F it, will just trim it off :/